roles/edpm_neutron_dhcp/molecule/default/verify.yml

```diff
@@ -51,3 +51,31 @@
             that:
               - dhcp_config.stat.exists
             fail_msg: "dhcp agent config file does not exist"
+
+    - name: Ensure that haproxy wrapper script was created
+      block:
+        - name: haproxy wrapper exists
+          become: true
+          ansible.builtin.stat:
+            path: "/var/lib/neutron/dhcp_agent_haproxy_wrapper"
+          register: haproxy_wrapper
+
+        - name: assert that haproxy wrapper exists
+          ansible.builtin.assert:
+            that:
+              - haproxy_wrapper.stat.exists
+            fail_msg: "haproxy wrapper script does not exist"
+
+    - name: Ensure that dnsmasq wrapper script was created
+      block:
+        - name: dnsmasq wrapper exists
```

Comment:
I believe names of tasks must start from upper-case (after lint-production enforcement lands)